src/Exizent.CaseManagement.Client/Models/EstateItems/BankAccountResourceRepresentationBase.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Exizent.CaseManagement.Client.Models.EstateItems;
+
+public class BankAccountResourceRepresentationBase : EstateItemResourceRepresentationBase, IStandardJointOwnership, IHasAddress, IRealisable
```

Comment:
should we abstract these so consumers are forced to use the GET/PUT/POST versions?